<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView_LoadImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_LoadImage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>179</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_LoadTrainImageData">
      <property name="text">
       <string>Load Train Image Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_LoadTrainLabelData">
      <property name="text">
       <string>Load Train Label Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_LoadTestImageData">
      <property name="text">
       <string>Load Test Image Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_LoadTestLabelData">
      <property name="text">
       <string>Load Test Label Data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
